<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ipw::RangeSelectWidget</class>
 <widget class="QWidget" name="ipw::RangeSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>139</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioRangeCustom">
       <property name="text">
        <string>Manual</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioRangeAuto">
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioRangeAll">
       <property name="text">
        <string>Everything</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ipw::HistogramPlot" name="histogram" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkHistogramLog">
     <property name="text">
      <string>logarithmic histogram</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QSlider" name="sliderMinMax1">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="singleStep">
        <number>10000</number>
       </property>
       <property name="pageStep">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="spinMinMax1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="spinMinMax2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="sliderMinMax2">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="singleStep">
        <number>10000</number>
       </property>
       <property name="pageStep">
        <number>100000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ipw::HistogramPlot</class>
   <extends>QWidget</extends>
   <header>ipw/histogramplot.h</header>
   <container>1</container>
   <slots>
    <slot>setScaleLog(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkHistogramLog</sender>
   <signal>clicked(bool)</signal>
   <receiver>histogram</receiver>
   <slot>setScaleLog(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
